app/models/role.rb

```diff
@@ -54,6 +54,14 @@ class Role < ActiveRecord::Base
   class_attribute :merge_excluded_attributes
   self.merge_excluded_attributes = []
 
+  FeatureGate.if('groups.nextcloud') do
```

Comment:
Good point.

Reply:
I was not able to extract this into a concern, despite following the usual pattern. Since it does not bother me too much personally, I will not do this now.